src/main/java/disa/notification/service/service/impl/MailServiceImpl.java

```diff
@@ -35,10 +40,20 @@ public class MailServiceImpl implements MailService {
 
     private TemplateEngine templateEngine;
     private final MessageSource messageSource;
+    
+    private final String startDateFormatted;
+    private final String endDateFormatted;
+    
+    @Autowired
+    private SeafileService seafileService;
```

Comment:
The project is already mostly using the constructor injection pattern and not property injection.